### DISCUSSION TITLE
User limit of 100 appears suddenly

### DISCUSSION THREAD

**Discussion by autoantwort** [nextcloud/all-in-one#5655] [Questions] [answered]
Hi, we are currently hitting the user limit of 100, but the strange thing is we are using nextcloud aio since months with over 300 users without getting any warning and error. Why do we the error now and not beforehand?

**Comment by szaimen** [comment, marked as answer]
Hi, I cannot tell why the limit only appears suddenly. It was since the beginning hard-coded to 100 users...

**Reply by autoantwort** [reply]
Hm. According to the code it does not count disabled users: https://github.com/nextcloud/server/blob/01b2ae4e4e47a58e3036184bbd52d8fdd76ee412/lib/private/Support/Subscription/Registry.php#L189 
Since https://\<domain\>/settings/users/disabled returns 325 Entries, we should be below this limit. Maybe I should create a bug report in the server repo

**Reply by szaimen** [reply]
> Maybe I should create a bug report in the server repo

Yes, sounds good
